sympy/solvers/diophantine.py

```diff
@@ -93,7 +93,8 @@ def _even(i):
     return i % 2 == 0
 
 
-def diophantine(eq, param=symbols("t", integer=True), syms=None):
+def diophantine(eq, param=symbols("t", integer=True), syms=None,
+                base_soln=True):
```

Comment:
I have added few line in this [commit](https://github.com/sympy/sympy/pull/11334/commits/dc4cfb145d439a4867a401d7f3d9f1dd3efb300e)

Thanks.